pkg/controller/migplan/migplan_controller.go

```diff
@@ -92,9 +94,11 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	err = c.Watch(
 		&source.Kind{Type: &migapi.MigCluster{}},
 		handler.EnqueueRequestsFromMapFunc(func(a client.Object) []reconcile.Request {
-			return migref.GetRequests(a, migapi.MigPlan{})
+			return migref.GetRequests(a, migapi.OpenshiftMigrationNamespace, migapi.MigPlan{})
```

Comment:
the last argument should be `migapi.MigCluster{}`?

Reply:
@alaypatel07 Nope, the last argument is correct. the event `a` coming from MigCluster is passed to `GetRequests`. The second parameter is the kind of the Owner not the target.